docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/ZAP.xml

```diff
@@ -58,6 +58,63 @@
     handling your support request, or you can upload the file into the issue
     tracking system. Remember, you should only use the email feature if
     appropriate to share your data.</para>
+
+    <sect3 id="ZAP_Config_Options">
+      <title>Configuration Options</title>
+
+      <para>The component logs are some of the most important artifacts
+      included in these ZAP reports. </para>
+
+      <para>The logging behavior differs on bare-metal and containerized
+      deployments. In bare metal, Some logs are always slated to be included
+      (Thormaster, ECLCC Server, and ECL Agent). The UI checkbox “Include
+      worker logs” controls the inclusion of the Thor worker (fka Thor slave)
+      log files. </para>
+
```

Comment:
there is no need to mention the former name. Also you use the Term tjhoprmaster here and later (line 93) you call it Thor manager. Thor Manager is preferred